pkg/bootstrapper/bootstrapper.go

```diff
@@ -522,6 +526,9 @@ func (wmcb *winNodeBootstrapper) generateInitialKubeletArgs(args map[string]stri
 	if nodeWorkerLabel, ok := args["node-labels"]; ok {
 		kubeletArgs = append(kubeletArgs, "--"+"node-labels"+"="+nodeWorkerLabel)
 	}
+	if wmcb.nodeIP != "" {
```

Comment:
Shouldn't we validate the `nodeIP` rather than blindly using it?

Reply:
Added it to the `newWinNodeBootstrapper` function